Add Localization functionality. Remove redundant files. Add /analytics/ endpoint.

sebastianpiresmolin opened the pull request:
This pull request includes significant changes to the `VizGurka` project, primarily focusing on renaming and restructuring pages, removing outdated code, and introducing localization support. Key changes include the renaming of the `Product` page to `Features`, removing the `Analytics` page, and adding localization to the `Features` page.

### Page Renaming and Restructuring:
* [`source/VizGurka/Pages/Features/Features.cshtml`](diffhunk://#diff-3baaf38704ff75d0d02d736c9c67f3dc168009148a4331ec987e81f3f5fefb2dL1-R13): Renamed from `Product.cshtml` and updated the route and layout. Adjusted references within the file to match the new naming convention. [[1]](diffhunk://#diff-3baaf38704ff75d0d02d736c9c67f3dc168009148a4331ec987e81f3f5fefb2dL1-R13) [[2]](diffhunk://#diff-3baaf38704ff75d0d02d736c9c67f3dc168009148a4331ec987e81f3f5fefb2dL22-R25) [[3]](diffhunk://#diff-3baaf38704ff75d0d02d736c9c67f3dc168009148a4331ec987e81f3f5fefb2dL43-R46) [[4]](diffhunk://#diff-3baaf38704ff75d0d02d736c9c67f3dc168009148a4331ec987e81f3f5fefb2dL139-L145)
* [`source/VizGurka/Pages/Features/Features.cshtml.cs`](diffhunk://#diff-5fb0f0ebf80cccd901d844464991a219994ac4ca7b0288954acb5bb37769cfd5R6-R22): Renamed from `Product.cshtml.cs` and updated the class name and namespace. Added properties and methods for localization. [[1]](diffhunk://#diff-5fb0f0ebf80cccd901d844464991a219994ac4ca7b0288954acb5bb37769cfd5R6-R22) [[2]](diffhunk://#diff-5fb0f0ebf80cccd901d844464991a219994ac4ca7b0288954acb5bb37769cfd5R35-R40) [[3]](diffhunk://#diff-5fb0f0ebf80cccd901d844464991a219994ac4ca7b0288954acb5bb37769cfd5R94)

### Removal of Outdated Code:
* [`source/VizGurka/Pages/Analytics/Index.cshtml`](diffhunk://#diff-fd24d3eb0c921cc9c1f4deee05e7f13056087da18474818f1e56948421e90929L1-L34): Removed the entire file as it is no longer needed.
* [`source/VizGurka/Pages/Analytics/Index.cshtml.cs`](diffhunk://#diff-c2ffcda2f887a5b86c9144f18e10a1cb49359826c9e049079899330ec6ca2c7aL1-L24): Removed the entire file as it is no longer needed.
* [`source/VizGurka/Pages/Analytics/Index.cshtml.css`](diffhunk://#diff-ea82602fb74527cac118b04a36cae0f2bc05cc641ca3ff1dba9429e131045f1aL1-L13): Removed the entire file as it is no longer needed.
* [`source/VizGurka/Pages/Features/Index.cshtml`](diffhunk://#diff-14ff15782d935c93ce5197ac36851403ce7da035a22c4d48834a7a9c9fc481feL1-L296): Removed the entire file as it is no longer needed.
* [`source/VizGurka/Pages/Features/Index.cshtml.cs`](diffhunk://#diff-5ab6b97376c7b266141764b66824d79111212827e8a4c7ef632e8a3714c4275fL1-L31): Removed the entire file as it is no longer needed.
* [`source/VizGurka/Pages/Features/Index.cshtml.css`](diffhunk://#diff-84fa801999c0eff2422280e165f367b74e42d4a10470ded568cf40f4407f45f6L1-L114): Removed the entire file as it is no longer needed.

### Localization Support:
* [`source/VizGurka/Pages/Features/Features.cshtml`](diffhunk://#diff-5fb0f0ebf80cccd901d844464991a219994ac4ca7b0288954acb5bb37769cfd5R6-R22): Added localization support using `IStringLocalizer` for various UI elements. [[1]](diffhunk://#diff-5fb0f0ebf80cccd901d844464991a219994ac4ca7b0288954acb5bb37769cfd5R6-R22) [[2]](diffhunk://#diff-5fb0f0ebf80cccd901d844464991a219994ac4ca7b0288954acb5bb37769cfd5R35-R40)

### Minor Changes:
* [`source/VizGurka/Pages/Analytics/Analytics.cshtml`](diffhunk://#diff-b6961d79ef0a8f77c96875e0cde89f725d4a69e1376f7b5413b21a85e81efbccR1-R9): Added a new page for analytics with a basic layout and model.
* [`source/VizGurka/Pages/Index.cshtml`](diffhunk://#diff-bfdde85cf3b1a56690f45267e542e8d42a786efe10482ed8a4bafed1449eb275R3): Added localization support by using `Microsoft.Extensions.Localization`.

skjohansen commented:
@sebastianpiresmolin  The text in the Localize properties defaults to Swedish, I think that the application should in first hand be English.

sebastianpiresmolin commented:
Changed the Localization properties so that English is now the Default. Removed the sv-SE.resx files and replaced them with en-GB.resx with corresponding string content changes.
